src/Restaurant.js

```diff
@@ -23,11 +23,9 @@ const styles = {
   var gitDomain = 'taylorakin.github.io';
 
   var currentDomain = document.domain;
-  console.log(currentDomain);
-
 
   if(currentDomain === gitDomain){
-    console.log('ta');
+  
     path = "/react_restaurant";
```

Comment:
You can use double quote or single quote for string.  It's better to be consistent. So choose one and use the same style for the whole project.

var gitDomain = 'taylorakin.github.io';
path = "/react_restaurant";

Reply:
Thanks